client/pluginmanager/csimanager/volume.go

```diff
@@ -159,52 +164,59 @@ func capabilitiesFromVolume(vol *structs.CSIVolume) (*csi.VolumeCapability, erro
 // stageVolume prepares a volume for use by allocations. When a plugin exposes
 // the STAGE_UNSTAGE_VOLUME capability it MUST be called once-per-volume for a
 // given usage mode before the volume can be NodePublish-ed.
-func (v *volumeManager) stageVolume(ctx context.Context, vol *structs.CSIVolume) (string, error) {
+func (v *volumeManager) stageVolume(ctx context.Context, vol *structs.CSIVolume) error {
 	logger := hclog.FromContext(ctx)
 	logger.Trace("Preparing volume staging environment")
-	stagingPath, isMount, err := v.ensureStagingDir(vol)
+	hostStagingPath, isMount, err := v.ensureStagingDir(vol)
 	if err != nil {
-		return "", err
+		return err
 	}
-	logger.Trace("Volume staging environment", "pre-existing_mount", isMount, "staging_path", stagingPath)
+	pluginStagingPath := v.stagingDirForVolume(v.containerMountPoint, vol)
+
+	logger.Trace("Volume staging environment", "pre-existing_mount", isMount, "host_staging_path", hostStagingPath, "plugin_staging_path", pluginStagingPath)
 
 	if isMount {
-		logger.Debug("re-using existing staging mount for volume", "staging_path", stagingPath)
-		return stagingPath, nil
+		logger.Debug("re-using existing staging mount for volume", "staging_path", hostStagingPath)
```

Comment:
Maybe worth having this field be `host_staging_path` as well, for folks who aren't running at `trace`.